src/Symfony/OtelSdkBundle/Resources/config/sdk.php

```diff
@@ -35,24 +38,24 @@
 
     // UTIL
 
-    $helper->setService(Trace\SystemClock::class);
+    $helper->setService(SystemClock::class);
```

Comment:
Maybe a side-issue for another time, but `setService` without a key implies that there is one, and repeated calls would replace it. `addService` instead?

Reply:
The key is generated from the FQCN, so while you could argue, that the class is indeed added, it cannot only be added once. It's just configuration. But feel free to change it, Brett. :)
```php
/**
     * @param string $class
     * @psalm-param class-string $class
     * @param bool $alias
     * @return ServiceConfigurator
     */
    public function setService(string $class, bool $alias = true): ServiceConfigurator
    {
        $id = ServiceHelper::classToId($class);
        $serviceConfigurator = $this->getConfigurator()->set($id, $class);
        if ($alias === true) {
            $this->setAlias($class, $id);
        }

        return $serviceConfigurator;
    }

```